### ISSUE TITLE
Control location of output files

### ISSUE THREAD

**Issue by tischi** [royerlab/ultrack#74]
I think each run creates a data.db and metadata.toml file?

Would it be possible to be able to explicitly specify in which directory those files are saved?

**Comment by JoOkuma** [comment]
Yes, this is the `config.data_config.working_dir` parameter, [more info here](https://github.com/royerlab/ultrack/tree/main/ultrack/config).
